src/TestFramework/MSTest.Core/Assertions/Assert.cs

```diff
@@ -1754,10 +1754,15 @@ public static void IsNotInstanceOfType(object value, Type wrongType, string mess
         /// </exception>
         public static void IsNotInstanceOfType(object value, Type wrongType, string message, params object[] parameters)
         {
-            if (wrongType == null || value == null)
+            if (wrongType == null)
```

Comment:
Why was this required ? Do we have an issue ?

Reply:
Yes. Also the functionality was clearly wrong.